fusedrug/data/tokenizer/modulartokenizer/multi_tokenizer.py

```diff
@@ -284,6 +295,47 @@ def load_from_jsons(tokenizers_info: List) -> Any:
         """
         return ModularTokenizer(tokenizers_info=tokenizers_info, load_adjusted_jsons=True)
 
+    @staticmethod
+    def load(path: str) -> Any:
+        """Reads all information that was saved by ModularTokenizer.save(), and creates a modular tokenizer based on it.
+
+        Args:
+            path: directory that contains a file named config.yaml which defines list of dictionaries containing the following:
+            [
+                   {
+                        "name": a name of a tokenizer
+                        "modular_json_path":out_path for tokenizer_type
+                    }
+            ]
+
+        Returns:
+            object: Loaded ModularTokenizer
+        """
+
+        def fix_json_paths(
+            loaded_conf: omegaconf.listconfig.ListConfig,
+            path: str,
+        ) -> omegaconf.listconfig.ListConfig:
+            """Since the path passed to ModularTokenizer.load() must contain all of the modular jsons and the config that defines their relations, all json
+            paths in the config must point to path. This function replaces any dirname of any json in the loaded config (found in path) with path.
+            Args:
+                loaded_conf (Union[List, omegaconf.listconfig.ListConfig]): _description_
+                path (str): _description_
```

Comment:
path to a dir that contains config.yaml, right?

Reply:
yes